presto-main/src/test/java/com/facebook/presto/operator/scalar/AbstractTestFunctions.java

```diff
@@ -54,7 +54,7 @@
 
 public abstract class AbstractTestFunctions
 {
-    private final Session session;
+    protected final Session session;
```

Comment:
maybe expose that with a method?

Reply:
`functionAssertions` is exposed as a field. I will leave this one too.